packages/block-library/src/button/edit.native.js

```diff
@@ -477,7 +477,7 @@ class ButtonEdit extends Component {
 						placeholderTextColor={
 							styles.placeholderTextColor.color
 						}
-						identifier="content"
+						identifier="text"
```

Comment:
I am not sure what that means. Could you please elaborate a bit?

Reply:
Crash message was saying:

`RichText needs an identifier prop that is the block attribute key of the attribute it controls. Its type is expected to be a string, but was a number`.

I've compared current `identifier` prop with web button and set the same value equal to `text`.